pkg/Microsoft.ML.Featurizers/Microsoft.ML.Featurizers.nupkgproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Additional ML.NET featurizers for AutoML</PackageDescription>
```

Comment:
Is `for AutoML` important enough to use in the description of what this package is?